config_src/nuopc_driver/mom_ocean_model_nuopc.F90

```diff
@@ -51,7 +51,7 @@ module MOM_ocean_model_nuopc
 use coupler_types_mod,       only : coupler_type_spawn, coupler_type_write_chksums
 use coupler_types_mod,       only : coupler_type_initialized, coupler_type_copy_data
 use coupler_types_mod,       only : coupler_type_set_diags, coupler_type_send_data
-use mpp_domains_mod,         only : domain2d, mpp_get_layout, mpp_get_global_domain
+use mpp_domains_mod,         only : domain2d, mpp_get_layout, mpp_get_global_domain,mpp_get_pelist
```

Comment:
mpp_get_pelist is no longer used? it needs to be removed here and in the commented out code at LN 445